clang/test/OpenMP/taskloop_strictmodifier_codegen.cpp

```diff
@@ -0,0 +1,256 @@
+// RUN: %clang_cc1 -verify -triple x86_64-apple-darwin10 -fopenmp -x c++ -emit-llvm %s -o - | FileCheck %s
```

Comment:
Can you auto generate the check lines?

Reply:
This file look like auto generated but there is no head line at the beginning of the file